qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -554,6 +581,11 @@ def open_settings_window(self):
         settings = SettingsPresenter(self)
         settings.show()
 
+    def open_settings_layout_window(self):
+        settings = SettingsPresenter(self)
+        settings.general_settings.focus_layout_box()
+        settings.show()
```

Comment:
For me the `new_layout_name` box isn't actually made visible by this function. This is what I see after clicking `View -> User Layouts -> Settings`:
![image](https://user-images.githubusercontent.com/46960113/60272897-e7319180-98ec-11e9-8537-3f9f20863664.png)

I think this is because Qt doesn't actually know how far down to scroll until the widgets are shown, so swapping lines 586 and 587 should help.

Reply:
On my screen half the `Layout Name` box is still hidden when showing the layout settings.
![image](https://user-images.githubusercontent.com/46960113/60329489-5c9e7000-9988-11e9-9d3c-0285ed966566.png)

 There's an optional `yMargin` argument in `ensureWidgetVisible`. From my testing setting this to 75 works on my machine (the default is 50).